### PR TITLE
[APP-2479] Remove old theme code and resources

### PR DESCRIPTION
**Pull request by G0retZ**
**What does this PR do?**

   Remove unused colours, fonts, resources and source code. Perform a general clean up in theme.

   In order to do so I had to refactor some things to decouple from old theme resources.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] By commits

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-2479](https://aptoide.atlassian.net/browse/APP-2479)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2479](https://aptoide.atlassian.net/browse/APP-2479)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[APP-2479]: https://aptoide.atlassian.net/browse/APP-2479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2479]: https://aptoide.atlassian.net/browse/APP-2479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Check.kt
Comment (GuiFPires) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Check.kt:1]: This one will be needed on payments

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Logout.kt
Comment (GuiFPires) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Logout.kt:1]: This one will be needed in payments

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Bug.kt
Comment (GuiFPires) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Bug.kt:1]: From the new UI, we should keep it.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Gift.kt
Comment (GuiFPires) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Gift.kt:1]: From the new UI, we should keep it.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/NoWifi.kt
Comment (GuiFPires) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/NoWifi.kt:1]: From the new UI, we should keep it.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt
Comment (GuiFPires) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt:1]: From the new UI, we should keep it.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/theme/Theme.kt
[the remark this answers is not in the document]
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/theme/Theme.kt:1]: Yes, the default one is not always an exactly Roboto and may change depending on the vendor firmware. Even mb changed by the user settings.
So instead for relying on that better explicitly specify the type face.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Check.kt
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Check.kt:1]: Yes

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt [hunk 1]
```diff
@@ -1,67 +0,0 @@
...
-  path(
-    fill = SolidColor(primary),
-  ) {
-    moveTo(23.2833f, 56.2834f)
-    lineTo(15.5833f, 48.4f)
-    curveTo(19.3722f, 44.6111f, 23.7111f, 41.7084f, 28.6f, 39.6917f)
-    curveTo(33.4889f, 37.675f, 38.6222f, 36.6667f, 44f, 36.6667f)
-    curveTo(49.3778f, 36.6667f, 54.5264f, 37.6903f, 59.4458f, 39.7375f)
-    curveTo(64.3653f, 41.7848f, 68.6889f, 44.7334f, 72.4167f, 48.5834f)
-    lineTo(64.7167f, 56.2834f)
```
Comment (phillipsgoncalves) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt:40]: Isn't this one necessary ?

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/PublisherTakeOverBundle.kt [hunk 2]
```diff
@@ -214,15 +211,14 @@ fun PublisherTakeOverListView(
...
-              style = AppTheme.typography.gameTitleTextCondensed
+              style = AGTypography.DescriptionGames
             )
           }
-          AppGamesButton(
-            title = "Install",
+          PrimarySmallButton(
             onClick = {},
-            style = Default(fillWidth = false),
+            title = "Install",
```
Comment (phillipsgoncalves) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/PublisherTakeOverBundle.kt:221]: Not related to the PR but is this string supposed to be hardcoded?

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_oos/OutOfSpaceAppItem.kt [hunk 3]
```diff
@@ -93,30 +91,24 @@ fun AppItem(
...
-    AppGamesOutlinedButton(
-      title = "Uninstall",
+    SecondarySmallOutlinedButton(
       onClick = action,
-      enabled = true,
-      style = Default(fillWidth = false),
-      modifier = Modifier
-        .wrapContentWidth()
-        .height(32.dp)
+      title = "Uninstall",
```
Comment (phillipsgoncalves) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_oos/OutOfSpaceAppItem.kt:111]: Not related to PR but is this string supposed to be hardcoded ?

**Comment by phillipsgoncalves** [comment]
Saw some deleted icons. I was trying to see if they were supposed to be deleted or not, but then saw that guilherme commented on some as well, so i am assuming he checked all

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt (hunk 1 above)
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt:40]: This one is not used and will not be used

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Wifi.kt:1]: This is not used and would not be used.
The related screen already uses a different image.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/NoWifi.kt
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/NoWifi.kt:1]: This is not used and would not be used.
The related screen already uses a different image.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Bug.kt
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Bug.kt:1]: This is not used and would not be used.
The related screen already uses a different image.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Gift.kt
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Gift.kt:1]: This is not used and would not be used.
The related screen already uses a different image in your PR.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Check.kt
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Check.kt:1]: True, fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Logout.kt
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Logout.kt:1]: True, fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_oos/OutOfSpaceAppItem.kt (hunk 3 above)
Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/feature_oos/OutOfSpaceAppItem.kt]: Fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/theme/AGTypography.kt [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  )
+  val TitleGames = TextStyle(
+    fontFamily = robotoFontFamily,
+    fontWeight = FontWeight.Medium,
+    fontSize = 20.sp,
+    lineHeight = 26.sp
+  )
+  val SubHeadingM = TextStyle(
+    fontFamily = robotoFontFamily,
+    fontWeight = FontWeight.Medium,
```
Comment (pedroagostinhoaptoide) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/theme/AGTypography.kt:90]: This font is actually bold on design, although being marked as 'Medium'

Comment (G0retZ) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/theme/AGTypography.kt]: Fixed 👍

**Review by GuiFPires — APPROVED** [review]
I feel like it's done :) Let's wait for the others reviews as well

**Review by phillipsgoncalves — APPROVED** [review]

**Review by pedroagostinhoaptoide — APPROVED** [review]

**Review by ricardo-frango — APPROVED** [review]

**Pull request merged**